### PR TITLE
Feature/prepare deployment

### PR DESCRIPTION
**Pull request by MichaelSchubertTNG**

### CODE REVIEW

File: backend/README.md
Comment (c7h) [inline backend/README.md:15]: with the flyway integration, this seems to be obsolete.

File: backend/local_deployment/docker-compose.yml
Comment (c7h) [inline backend/local_deployment/docker-compose.yml:8]: we should put this in an environment file and create a environment.template file. This way, chances are lower to forget changing this for production.

File: backend/local_deployment/docker-compose.yml
Comment (c7h) [inline backend/local_deployment/docker-compose.yml:25]: see comment above.

File: backend/local_deployment/docker-compose.yml
Comment (c7h) [inline backend/local_deployment/docker-compose.yml]: Would the container name `db` work here? In docker, it should resolve to the internal docker ip for the database server.

**Review by c7h — CHANGES_REQUESTED** [review]
Minor changes in the docker compose file and some questions about README.

File: backend/local_deployment/docker-compose.yml
Comment (MichaelSchubertTNG) [reply backend/local_deployment/docker-compose.yml]: Yes that works, so then we can get rid of the network_mode as well, thank you!

File: backend/README.md
Comment (MichaelSchubertTNG) [reply backend/README.md:15]: No, flyway acts on the initialized betten database

File: backend/local_deployment/docker-compose.yml
Comment (MichaelSchubertTNG) [reply backend/local_deployment/docker-compose.yml:8]: This docker-compose is explicitly only for the local setup. The productive deployment will be with kubernetes

**Comment by atomdampflok** [comment]
don't merge right now, I messed something up with the commits

**Review by c7h — APPROVED** [review]

**Pull request merged**
